navidrome/egg-navidrome.json

```diff
@@ -0,0 +1,142 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-12-27T22:06:30+01:00",
+    "name": "Navidrome",
+    "author": "felixseiboldt@gmail.com",
+    "description": "Navidrome Music Server - A lightweight music server for your personal collection.",
+    "features": null,
+    "docker_images": {
+        "devilapi\/navidrome-pterodactyl:latest": "devilapi\/navidrome-pterodactyl:latest"
+    },
+    "file_denylist": [],
+    "startup": "\/bin\/bash \/home\/container\/start.sh",
+    "config": {
+        "files": "{\r\n}",
+        "startup": "{\r\n    \"done\": \"Navidrome is ready to accept requests\"\r\n}",
+        "logs": "{\r\n    \"logs\": {\r\n        \"stdout\": \"\/home\/container\/logs\/stdout.log\",\r\n        \"stderr\": \"\/home\/container\/logs\/stderr.log\"\r\n    }\r\n}",
+        "stop": "^C"
+    },
+    "scripts": {
+        "installation": {
+            "script": "mkdir \/mnt\/server\/music",
```

Comment:
Based on this I assume the files are in the image. This is not allowed. The install script should download them. The image should only hold pacakges.